cmd/soroban-rpc/internal/db/transaction.go

```diff
@@ -107,8 +107,7 @@ func (txn *transactionHandler) InsertTransactions(lcm xdr.LedgerCloseMeta) error
 	}
 	_, err = query.RunWith(txn.stmtCache).Exec()
 
-	L.WithError(err).
```

Comment:
why is the error removed from the log?

Reply:
Because it can be nil (it normally is nil) and the consumer already gets the error.